example/template/ruby-rails-example.json

```diff
@@ -1,171 +1,213 @@
+            "intervalSeconds": 1,
+            "timeoutSeconds": 120
+          },
+          "resources": {}
+        },
+        "triggers": [
+          {
+            "type": "ImageChange",
+            "imageChangeParams": {
+              "automatic": true,
+              "containerNames": [
+                "rails-blog"
+              ],
+              "from": {
+                "name": "origin-rails-blog"
+              },
+              "lastTriggeredImage": ""
+            }
+          }
+        ],
+        "replicas": 1,
+        "selector": {
+          "name": "frontend"
+        },
+        "template": {
+          "metadata": {
+            "creationTimestamp": null,
             "labels": {
               "name": "frontend"
             }
           },
-          "replicaSelector": {
-            "name": "frontend"
-          },
-          "replicas": 1
-        }
-      },
-      "triggers": [
-        {
-          "imageChangeParams": {
-            "automatic": true,
-            "containerNames": [
-              "rails-blog"
+          "spec": {
+            "containers": [
+              {
+                "name": "rails-blog",
+                "image": "origin-rails-blog",
+                "ports": [
+                  {
+                    "containerPort": 8080,
+                    "protocol": "TCP"
+                  }
+                ],
+                "resources": {},
+                "terminationMessagePath": "/dev/termination-log",
+                "imagePullPolicy": "IfNotPresent",
+                "capabilities": {},
+                "securityContext": {
+                  "capabilities": {},
+                  "privileged": false
+                }
+              }
             ],
-            "from": {
-              "name": "origin-rails-blog"
-            },
-            "tag": "latest"
-          },
-          "type": "ImageChange"
+            "restartPolicy": "Always",
+            "dnsPolicy": "ClusterFirst"
+          }
         }
-      ]
+      },
+      "status": {}
     }
   ],
-  "kind": "Template",
-  "metadata": {
-    "annotations": {
-      "description": "This example shows how to create a simple ruby on rails application in openshift origin v3",
-      "tags": "instant-app,ruby,rails",
-      "iconClass" : "icon-ruby"
-    },
-    "name": "rails-blog-sample"
-  },
   "parameters": [
     {
-      "description": "GitHub URL",
       "name": "GITHUB_URL",
+      "description": "The URL with your Rails application source code.",
       "value": "git://github.com/ewolinetz/ruby-rails-example.git"
     },
     {
-      "description": "production secret key base",
-      "from": "[a-z0-9]{127}",
+      "name": "SECRET_KEY_BASE",
+      "description": "Your secret key for verifying the integrity of signed cookies.",
       "generate": "expression",
-      "name": "SECRET_KEY_BASE"
+      "from": "[a-z0-9]{127}"
     }
   ],
   "labels": {
     "template": "application-template-stibuild"
   }
-}
+}
```

Comment:
Just a UNIX convention, files end with "\n" (that's why GitHub, git and other tools will warn about it).
But I see you added one line too much, now it ends with an empty line "\n\n"...